discovery/http/http_test.go

```diff
@@ -162,3 +162,241 @@ func TestContentTypeRegex(t *testing.T) {
 		})
 	}
 }
+
+func TestSourceDisappeared(t *testing.T) {
+	var stubResponse string
+	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Set("Content-Type", "application/json")
+		fmt.Fprintln(w, stubResponse)
```

Comment:
I just wonder if that will work or if we need pointers.

Reply:
should I change the testcase?